NewForce-Cohort-4/tabloid-full-stack-team-magenta
# Ticket #8 Edit Posts

# Description
As a author I would like to be able to modify my Posts so that I can correct mistakes or add additional content.
​
# Type of change

#  Testing Instructions 

Pull down and checkout branch vo-posts
Start both server side in VS and and client side from the client folder
use foo@bar.com and password: ******** to log in to the application and see the home page
Once logged in navigate to the "Posts" or "My Posts" tab in the menubar. For posts that the logged in user is the author, there is a new edit button.
The edit button also appears in the details view when reading a post the user authored.
Clicking the edit button brings up a new edit page with the data filled in (this may take a minute, but should work when pages is loaded. Refreshing may cause unexpected bugs).
User can then change the data in the available fields and submit their changes. If you save changes, you should be redirected back to the details page for that updated post.
If you did not save changes you will be redirected back to the home list of posts. 

​
# Checklist:
​
